openqa-trigger-bisect-jobs

```diff
@@ -299,10 +304,17 @@ def main(args):
             ]
         )
 
-        out = openqa_clone(
-            params,
-            args.dry_run,
-        )
+        try:
+            out = openqa_clone(
+                params,
+                args.dry_run,
+            )
+        except subprocess.SubprocessError as err:
+            if 'the repositories for the below updates are unavailable' in str(err.stderr):
+                sys.exit(0)
```

Comment:
I am not sure about that but as I start getting a better view, I am wonder why we need to exit and not just log and continue. no need to answer. just wonder

Reply:
because we will get the same error message for every item in the loop. there is no sense in continuing if we don't schedule any bisecting jobs